integration-tests/downscope/downscope_test.go

```diff
@@ -105,23 +109,24 @@ func TestDownscopedToken(t *testing.T) {
 
 	for _, tt := range downscopeTests {
 		t.Run(tt.name, func(t *testing.T) {
-			err := helper(tt)
+			err := helper(t, tt)
 			// If a test isn't supposed to fail, it shouldn't fail.
 			if !tt.expectError && err != nil {
 				t.Errorf("test case %v should have succeeded, but instead returned %v", tt.name, err)
 			} else if tt.expectError && err == nil { // If a test is supposed to fail, it should return a non-nil error.
-				t.Errorf(" test case %v should have returned an error, but instaed returned nil", tt.name)
+				t.Errorf(" test case %v should have returned an error, but instead returned nil", tt.name)
 			}
 		})
 	}
 }
 
 // I'm not sure what I should name this according to convention.
-func helper(tt downscopeTest) error {
+func helper(t *testing.T, tt downscopeTest) error {
```

Comment:
nit: This name should still needs to be updated to describe what this function is doing.